### PR TITLE
Fixes MySQL export for more than one entry 

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fixes the export of more than one entry through the connection option #685 
Added ability to add server specific connection parameters in dbStrings, because MySQL and PostreSQL parameter names differ.

Using setCatalog instead of use database (recommended in mysql doc see: https://dev.mysql.com/doc/connector-j/en/connector-j-reference-configuration-properties.html , Section  "Initial Database for Connection" )

### CODE REVIEW

**Comment by oscargus** [comment]
Thanks for this PR!

Ideally, I guess, one would like to set parameters pairwise as `dbStrings.addParameter("allowMultiQueries", "true")` and then have `dbStrings.getParameters()` form the correct connection string (see e.g. https://github.com/JabRef/jabref/blob/master/src/main/java/net/sf/jabref/logic/net/URLDownload.java ). However, I assume that this may be the only parameter of interest to add...

**Comment by lenhard** [comment]
@Siedlerchr could you rebase this on master? There is also a good chance that this fixes your failing tests.

**Comment by Siedlerchr** [comment]
@lenhard  Yes, I wil check it this evening.

File: src/main/java/net/sf/jabref/sql/exporter/MySQLExporter.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 final public class MySQLExporter extends DBExporter {
 
     private static MySQLExporter instance;
+    private static String optAllowMultiQueries = "?allowMultiQueries=true";
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/sql/exporter/MySQLExporter.java]: Please make this final and use UPPER_CASE_NAMING.

**Comment by Siedlerchr** [comment]
I have recreated the PR on the current upstream master branch, so ignore this one here.
(I'm still figuring out that whole git mergin stuff)

**Pull request closed without merging**